### PR TITLE
Add improvements to the menu

### PR DESCRIPTION
**Pull request by mariano-aguero**
Look like this ..
Peek 03-03-2021 00-21

### CODE REVIEW

File: src/components/layouts/base/menues/connectedDropdown.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <v-list-item to="/supplyBorrow">
+      <v-list-item-icon>
+        <v-icon>mdi-bank</v-icon>
+      </v-list-item-icon>
+      <v-list-item-title>Supply/Borrow</v-list-item-title>
+    </v-list-item>
+
+    <v-list-item to="/status">
+      <v-list-item-icon>
+        <v-icon>mdi-frequently-asked-questions</v-icon>
```
Comment (pmprete) [inline src/components/layouts/base/menues/connectedDropdown.vue:26]:
```suggestion
        <v-icon>mdi-view-list-outline</v-icon>
```
faq icon already used that

**Review by pmprete — DISMISSED** [review]
Except my comment looks good, nice break up in components 
👍

File: src/components/layouts/base/menues/connectedDropdown.vue (hunk 1 above)
Comment (mariano-aguero) [reply src/components/layouts/base/menues/connectedDropdown.vue]: I made an update with the store icon, if you don't agree you can suggest one from the following list https://material.io/resources/icons/?icon=store&style=baseline

Comment (fgregoribattista) [reply src/components/layouts/base/menues/connectedDropdown.vue]: sounds good

**Review by fgregoribattista — APPROVED** [review]
it's ok for me

**Pull request merged**